src/app/views/account/account.component.html

```diff
@@ -89,7 +89,7 @@ <h2 class="col-9">myTOA</h2>
           <div mdcSubtitle1 class="mt-1">{{ 'pages.account.api_card.your_key' | translate }}</div>
           <code dir="ltr">{{ user.apiKey || 'Not found' }}</code>
           <div mdcSubtitle1 class="mt-2">{{ 'pages.account.api_card.docs' | translate }}
-            <a href="https://theorangealliance.org/apidocs/" target="_blank" (click)="this.sendAnalytic('leaving', '/account', 'api docs');">{{ 'general.here' | translate }}</a>.
+            <a routerLink="apidocs" target="_blank" (click)="this.sendAnalytic('leaving', '/account', 'api docs');">{{ 'general.here' | translate }}</a>.
```

Comment:
I meant to link to this: https://github.com/orange-alliance/the-orange-alliance/blob/master/src/app/app.component.html#L71

I believe that the `/` before `apidocs` is needed.

Reply:
You can see a proper use of `routerLink` there.

Reply:
Just made another set of changes -- think I got it right this time